SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1

```diff
@@ -79,46 +116,66 @@ function Set-Secret
         [Parameter(ValueFromPipelineByPropertyName)]
         [hashtable] $AdditionalParameters
     )
-    if($Secret -is [string]) {
-        $Secret = @{
-            URL = "http://sn"
```

Comment:
I am 99% sure that the default URL scheme of http://sn is required at all but I'll review this tomorrow just in case.